### DISCUSSION TITLE
슬슬 업로드도 해봐야지?

### DISCUSSION THREAD

**Discussion by RimWorldKorea** [RimWorldKorea/RMK#339] [일반]
원래는 git이라는 프로그램이 따로 있는데

콘솔창이나 리눅스에 익숙하지 않으면 써먹기가 힘들어서

좀 더 단순화하고 껍데기를 이쁘게 씌운 [GitHub Desktop](https://desktop.github.com/)이란게 있음

일단 다운받아서 한 번 둘러봐

@zilrty

**Comment by zilrty** [comment]
갸아악

**Reply by RimWorldKorea** [reply]
깔았으면 repository 설정해서 네 컴퓨터에 클론해봐

깃헙 파일을 그대로 컴퓨터에 복사하는 개념임

경로는 난 로컬모드 폴더에 해둠

branch는 bus로 설정

**Reply by zilrty** [reply]
하니까 로컬폴더에 RMK 생김

**Reply by RimWorldKorea** [reply]
이제 그 RMK 폴더에 네가 무슨 데이터를 변경하거나 하면
그 '내역'이 전부 너의 '로컬' 저장소(=네 컴퓨터의 RMK 폴더)에 기록됨
깃헙 데스크탑 열면 왼쪽에 체크박스 설정할 수 있는 목록 막 뜨는거 있을거임

그 내역을 체크하고 내용을 적은 뒤 아래의 commit to bus 버튼을 누르면
네 '로컬' 저장소에서 그 내역을 확정된 변경 내용으로 취급하는거임

커밋을 올리면 상단에 버튼 하나가 바뀌면서 push to origin인가로 바뀔건데
그 버튼을 누르면 네가 커밋한 변경내역이 '원격' 저장소(=깃헙의 RMK bus 브랜치)에 반영되는거임

원래는 bus 브랜치를 개인 저장소에 포크하는 과정이 필요한 걸로 아는데
collaborator는 원격 저장소에 바로 접근할 수 있어서 과정이 좀 다를거임

나중에 번역 작업할 일 있으면 시험삼아서 한 번 해봐
뭔 일 있으면 내가 취소하면 되니깐 편하게 해도 됨

**Comment by zilrty** [comment]
![제목 없음](https://github.com/RimWorldKorea/RMK/assets/132768180/9022e56a-6f0a-4585-adf5-78c1cab66dbb)

키에엑 이거 뭐임

**Reply by RimWorldKorea** [reply]
아 코드 오너만 직접 되나?

<img width="108" alt="화면 캡처 2024-01-04 051536" src="https://github.com/RimWorldKorea/RMK/assets/150018627/21daca52-b402-4116-86fb-d359bce198fb">

깃헙 홈페이지에서 `bus` 브랜치를 네 깃헙 원격 저장소로 포크한 뒤
그걸 깃헙 데스크탑을 통해 네 컴퓨터에 클론해서 거기다가 해봐

**Reply by RimWorldKorea** [reply]
<img src="https://github.com/RimWorldKorea/RMK/assets/150018627/042652fb-2ac1-413d-86e5-9f38522c9c34" />

이 흐름도에서 업로더에서 bus로 흐르는 화살표에 사실

![제목 없음](https://github.com/RimWorldKorea/RMK/assets/150018627/6ca52230-57d3-4ded-954d-f7424879bf2a)

이런 작업이 생략돼있다고 보면 됨

**Comment by RimWorldKorea** [comment]
@zilrty 

이거 매뉴얼 쓸때 써먹게 중간중간에 스샷도 좀 찍어줄 수 있음?

**Reply by zilrty** [reply]
아직 헷갈리는 상태라 좀 적응되면 찍어줘도 됨?

지금도 적어준거 보면서 이게 맞나 눌러보고 되면 아 맞네 이러고 있는 상태라 ㅋㅋ
